### PR TITLE
Increase max_length on field user_agent to 512 chars

### PR DESCRIPTION
**Pull request by mvasilchenko**
Hi, I'm noticed here, how are users trying to reset their passwords from several not-standard browsers, like Instagram browser and Facebook browser. And in this case, I am catching exception ```value too long for type character varying(256)```

This is an example Instagram browser user agent
```
'Mozilla/5.0 (Linux; Android 10; RMX2040 Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/91.0.4472.120 Mobile Safari/537.36 Instagram Android (29/10; 320dpi; 720x1456; realme; RMX2040; RMX2040; mt6768; ru_RU; 302733769)'
```

### CODE REVIEW

File: django_rest_passwordreset/models.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def generate_key():
         blank=True,
         null=True,
     )
-    user_agent = models.CharField(
-        max_length=256,
+    user_agent = models.TextField(
```
Comment (nezhar) [inline django_rest_passwordreset/models.py:66]: This should remain a `CharField`

Comment (mvasilchenko) [reply django_rest_passwordreset/models.py:66]: Yep, i'm already fix it, in last commits

Comment (kmasuhr) [reply django_rest_passwordreset/models.py:66]: @mvasilchenko Can you correct `TextField` in your PR? :)

**Comment by replygirl** [comment]
What's the blocker to getting this in?

**Comment by nezhar** [comment]
Closed by https://github.com/anexia-it/django-rest-passwordreset/pull/182

**Pull request closed without merging**